test/parallel/test-tls-interleave.js

```diff
@@ -31,7 +31,7 @@ const tls = require('tls');
 
 const fs = require('fs');
 
-const dir = common.fixturesDir;
+const dir = common.fixtures;
```

Comment:
Can you please follow @pawelgolda's suggestion and change this to `const fixtures = require('../common/fixtures');`? Thank you.